applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot/json_protocol_names_aws_iot.h

```diff
@@ -36,6 +36,7 @@
 #define DATA_TEMPERATURE    "temp"
 #define DATA_HUMIDITY	    "hum"
 #define DATA_PRESSURE       "atmp"
+#define DATA_AIR_QUALITY    "airq"
```

Comment:
```suggestion
#define DATA_BSEC_IAQ    "bsec_iaq"
```

This is a vendor specific unit, so mark it clearly.